activation/handler.go

```diff
@@ -493,6 +496,9 @@ func (h *Handler) HandleGossipAtx(ctx context.Context, peer p2p.Peer, msg []byte
 			log.Err(err),
 		)
 	}
+	if errors.Is(err, errKnownAtx) && peer == h.local {
+		return nil
+	}
```

Comment:
What happens on remote nodes if they receive the re-gossiped ATXs? Will this cause all nodes to keep logging warnings / errors about known ATXs?

Reply:
the expected behavior that they will drop them without gossiping further.
my hope was that we don't log them every time, i will check that it is the case.

Reply:
code doesn't spam, so neighbor will simply drop atx if it exists already